src/test/java/org/jenkinsci/plugins/workflow/steps/scm/SCMStepTest.java

```diff
@@ -186,16 +197,26 @@ public InvalidChangelogSCM() { }
         @Override public void checkout(Run<?,?> build, Launcher launcher, FilePath workspace, TaskListener listener, File changelogFile, SCMRevisionState baseline) throws IOException, InterruptedException {
             // Unlike the superclass, which adds an XML header to the file, we just ignore the file.
         }
-        @Override public ChangeLogParser createChangeLogParser() {
-            return new ChangeLogParser() {
-                public ChangeLogSet<? extends ChangeLogSet.Entry> parse(Run build, RepositoryBrowser<?> browser, File changelogFile) throws IOException, SAXException {
-                    Jenkins.XSTREAM2.fromXML(changelogFile);
-                    throw new AssertionError("The previous line should always fail because the file is empty");
-                }
-            };
-        }
+        ChangeLogParser changeLogParser = createChangeLogParser();
         @TestExtension("scmParsesUnmodifiedChangelogFile")
         public static class DescriptorImpl extends NullSCM.DescriptorImpl { }
     }
 
+    public static class FakeChangeLogSCM extends NullSCM {
+        @DataBoundConstructor
+        public FakeChangeLogSCM() { }
+        ChangeLogParser changeLogParser = createChangeLogParser();
+        @TestExtension("scmParsesFakeChangeLogSCMChangelogFile")
+        public static class DescriptorImpl extends NullSCM.DescriptorImpl { }
+    }
+
+    public ChangeLogParser createChangeLogParser() {
```

Comment:
If I add `@Override` here, which mimics [the way it used to be](https://github.com/kshultzCB/workflow-scm-step-plugin/pull/1/files#diff-862fb161c96b2514ed6613245a629e71L189), I get `Method does not override method from its superclass`.

Reply:
This does not get `@Override` because it is not being defined on a class that extends `NullSCM`. 
This is a perfect example of why to use `@Override` - the error is telling you something isn't where you expect it. 

Also, note did you say you wanted this method to be `static`?